server/src/handlers/http.rs

```diff
@@ -229,6 +230,10 @@ pub fn configure_routes(cfg: &mut web::ServiceConfig) {
                         .wrap(DisAllowRootUser),
                 ),
         );
+
+    let llm_query_api =
+        web::scope("/llm").service(web::resource("").route(web::post().to(llm::make_llm_request)));
```

Comment:
do we have permissions set up for this API - who can call this API?

Reply:
Hi, missed permissions I'm afraid. 
Rn anyone is able to call the API.
Will modify this to need the authorization base64 string being passed in header.